Platforms/QemuQ35Pkg/Library/EmuVariableFvbLib/EmuVariableFvbLib.inf

```diff
@@ -0,0 +1,43 @@
+## @file
+#  OVMF platform customization for EMU Variable FVB driver
+#
+#  This library handles hooks for the EMU Variable FVB driver.
+#
+#  Copyright (c) 2006 - 2018, Intel Corporation. All rights reserved.<BR>
+#
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+#
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = EmuVariableFvbLib
+  FILE_GUID                      = 8a6062ed-7140-4a74-b4ea-fe900e79e24b
+  MODULE_TYPE                    = BASE
+  VERSION_STRING                 = 1.0
+  LIBRARY_CLASS                  = PlatformFvbLib|DXE_RUNTIME_DRIVER

```

Comment:
I'd recommend calling this library instance `PlatformFvbLibEmu`.

1. It makes it clear that it is an instance of the `PlatformFvbLib` library class.
2. FVB is technically not restricted to variable services.

(2) is less important and not always true. (1) is the main reason I suggest the rename.